docs/tutorials/action_recognition/distributed_training.py

```diff
@@ -0,0 +1,153 @@
+and is not friendly for new comers and students to work on video related problems.
+
+Using distributed training is a natural choice. Spreading the huge computation over multiple machines can speed up training
+a lot. However, only a few open sourced Github repositories on video understanding support distributed training,
+and they often lack documentation for this feature.
+Besides, there is not much information/tutorial online on how to perform distributed training for deep video models.
+
+Hence, we provide a simple tutorial here to demonstrate how to use our code to perform distributed training of SlowFast models
+on Kinetics400 dataset.
+
+"""
+
+########################################################################
+# Distributed training
+# --------------------
+#
+# There are two ways in which we can distribute the workload of training a neural network across multiple devices,
+# data parallelism and model parallelism. Data parallelism refers to the case where each device stores a complete copy of the model.
+# Each device works with a different part of the dataset, and the devices collectively update a shared model.
+# When models are so large that they don't fit into device memory, then model parallelism is useful.
+# Here, different devices are assigned the task of learning different parts of the model.
+# In this tutorial, we describe how to train a model with devices distributed across machines in a data parallel way.
+#
+# There are some key concepts in distributed training, such as server, worker, scheduler, kvstore, etc.
+# Server is a node to store model's parameters and communicate with workers.
+# Worker is a node actually performing training on a batch of training samples.
+# Before processing each batch, the workers pull weights from servers.
+# The workers also send gradients to the servers after each batch.
+# Scheduler is to set up the cluster for each node to communicate.
+# Kvstore, which is key-value store, is a critical component used for multi-device training.
+# It stores model parameters and optimizers, to receive gradient and update model.
+# In order to keep this tutorial concise, I wouldn't go into details.
+# Readers can refer to MXNet `official documentation <https://mxnet.apache.org/api/faq/distributed_training>`_ for more information.
+
+
+########################################################################
+# How to use our code to train a SlowFast model in a distributed manner?
+# ----------------------------------------------------------------------
+#
+# In order to perform distributed training, you need to install MXNet and prepare the cluster ready.
+# MXNet provides a script ``tools/launch.py`` to make it easy to launch distributed training on a cluster with ssh, mpi, sge or yarn.
+
+################################################################
+# First, let's install MXNet.
+# ::
+#
+#     pip install mxnet-cu100
```

Comment:
A link to gluoncv installation guide might be useful here, since there are multiple versions available for mxnet